src/components/Grid.tsx

```diff
@@ -406,11 +424,26 @@ export default class Grid extends React.Component<
     return this.state.activeScheme;
   }
 
+  changelogModal() {
```

Comment:
Is this used anywhere?

Reply:
Yeah it was for testing purposes, cleaned it up afterwards.